SW/main/pot_detection.py

```diff
@@ -0,0 +1,43 @@
+from azure.cognitiveservices.vision.customvision.training import CustomVisionTrainingClient
+from azure.cognitiveservices.vision.customvision.training.models import ImageFileCreateEntry, Region
+from azure.cognitiveservices.vision.customvision.prediction import CustomVisionPredictionClient
+
+class PotDetector:
+
+    def __init__(self):
+        self.ENDPOINT = "https://southcentralus.api.cognitive.microsoft.com"
+
+        # Project keys
+        self.training_key = "dae03cb013f840658708cd62781d90c1"
+        self.prediction_key = "6212b24516c6492190c63d2b32084079"
+        self.project_id = "a97fb679-77e7-4e07-b946-81c752ee3112"
+
+        self.probability_min = 60
+
+        # Now there is a trained endpoint that can be used to make a prediction
+
+        self.predictor = CustomVisionPredictionClient(self.prediction_key, endpoint=self.ENDPOINT)
+
+    def update(self,image_path, stove):
+        # Open the sample image and get back the prediction results.
+        with open(image_path, mode="rb") as test_data:
```

Comment:
this should be refactored so image_path is instead the actual picture passed in (so each frame, the picture only has to be loaded once.)

Also, rename "test_data" to frame

Reply:
Do you mean just rename image_path to image or something?

Reply:
no. here, you are opening the image (With open(image_path)). That opens the image file. Instead of doing it here, do it in the main file. Pass in the image object.

Reply:
I see.